src/uniprotkb/components/protein-data-views/VariationView.tsx

```diff
@@ -212,12 +204,15 @@ const VariationView = ({
     !filteredVariants
   ) {
     return (
-      <div>
+      <section
+        style={{ marginInlineEnd: '1.2rem' }}
```

Comment:
This style also listed at line 487 so what do think about defining in a css file?

Reply:
I've now created a class for that